examples.md

```diff
@@ -108,27 +120,20 @@ steps:
 We cache the elements of the Cabal store separately, as the entirety of `~/.cabal` can grow very large for projects with many dependencies.
```

Comment:
Let's update this description, since we're no longer caching these separately.

Reply:
My understanding is that we are still caching them separately, just in one step now. I think we should keep it so that users don't accidentally cache the entire `~/.cabal` directory and use up all the cache storage.